src/main/kotlin/org/opensearch/replication/action/stop/TransportStopIndexReplicationAction.kt

```diff
@@ -68,7 +70,7 @@ class TransportStopIndexReplicationAction @Inject constructor(transportService:
                                                               IndexNameExpressionResolver,
                                                               val client: Client,
                                                               val replicationMetadataManager: ReplicationMetadataManager) :
```

Comment:
Can we add a comment here explaining that the stop replication action class in moved to common utils and link this PR here?